roop/swapper.py

```diff
@@ -52,7 +53,8 @@ def process_frames(source_path: str, temp_frame_paths: List[str], progress=None)
         temp_frame = cv2.imread(temp_frame_path)
         try:
             result = process_faces(source_face, temp_frame)
-            cv2.imwrite(temp_frame_path, result)
+            cv2.imwrite(state.get_frame_processed_name(temp_frame_path), result)

```

Comment:
It would make send to add a variable that points to state.get_frame_processed_name(temp_frame_path) so the reader can easier understand what it contains

Reply:
done